app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -279,10 +291,56 @@ public int onStartCommand(Intent intent, int flags, int startId) {
             createIVButton();
             createArcPointer();
             createArcAdjuster();
+            /* Assumes MainActivity initialized ScreenGrabber before starting this service. */
```

Comment:
That sounds... potentially fishy.  Shouldn't we call some `getInstance()` and have it fail if no initialization had been done?